client/a8c-for-agencies/sections/referrals/common/referral-details-table/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { DATAVIEWS_TABLE } from 'calypso/a8c-for-agencies/components/items-dashboard/constants';
+import { DataViews } from 'calypso/components/dataviews';
+import type { ReferralPurchase } from '../../types';
+import type { DataViewsColumn } from 'calypso/a8c-for-agencies/components/items-dashboard/items-dataviews/interfaces';
+
+import './style.scss';
+
+interface Props {
+	heading: string;
+	items: ReferralPurchase[]; // Update this when we have more types
+	fields: DataViewsColumn[];
+}
+
+export default function ReferralDetailsTable( { heading, items, fields }: Props ) {
+	return (
+		<div className="referrals-details-table__container redesigned-a8c-table">
+			<div className="referrals-details-table__heading">{ heading }</div>
+			<DataViews
```

Comment:
I didn't realize this... Can we use the `ItemsDataViews` here? Instead of directly the `DataViews` component?

Reply:
Mmm oh... forget about it, I thought this was the main table list. I think it's fine to reuse the DataViews component here... (I'm guessing...) I'm not sure if it's too much for the use case, but if it works, perfect.

Reply:
This was just a plain table. That's the reason I switched back to DataViews. 

But, I think it makes sense. I have made the changes: https://github.com/Automattic/wp-calypso/pull/91183/commits/d067245722c004c41a17dcba0505a902c3f286a8